### PR TITLE
fixes #13134 - Use patternfly icons

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by dLobatog** [comment]
@stbenjam  `job_invocations/_tab_hosts` and `_form` also use glyphicons

**Comment by stbenjam** [comment]
Thanks, updated

File: app/helpers/remote_execution_helper.rb [hunk 1]
```diff
@@ -60,19 +60,19 @@ def host_counter(label, count)
...
           else
-            content_tag(:i, '&nbsp'.html_safe, :class => 'glyphicon glyphicon-exclamation-sign') + content_tag(:span, _('failed'), :class => 'status-error')
+            icon_text('error-circle-o', _('failed'), :kind => 'pficon')
           end
         when 'success'
-          content_tag(:i, '&nbsp'.html_safe, :class => 'glyphicon glyphicon-ok-sign') + content_tag(:span, _('success'), :class => 'status-ok')
+          icon_text('ok', _('success'), :kind => 'pficon')
         when 'pending'
-          content_tag(:i, '&nbsp'.html_safe, :class => 'glyphicon glyphicon-question-sign') + content_tag(:span, _('pending'))
+          icon_text('question', _('pending'), :kind => 'fa')
```
Comment (iNecas) [inline app/helpers/remote_execution_helper.rb]: The question mark on pending might be misleading for somebody (or the whole pending word), running would be better probably (but not sure I like the patterfly running icon), anyway, not necessary to be done as part of this PR

**Comment by iNecas** [comment]
one comment, but necesary needed to do on this PR, tested, ACK

File: app/helpers/remote_execution_helper.rb (hunk 1 above)
Comment (stbenjam) [inline app/helpers/remote_execution_helper.rb]: I agree, do you want to file a separate issue? The question mark itself looks like a possible error as well, certainly could be confusing.

Comment (iNecas) [inline app/helpers/remote_execution_helper.rb]: +1 for an issue

**Pull request merged**